README.md

```diff
@@ -612,19 +612,26 @@ Put this into your `$HOME/.zshrc` to call `nvm use` automatically whenever you e
 ```zsh
 # place this after nvm initialization!
 autoload -U add-zsh-hook
+find-old-nvmrc() {
+  local path_
+  path_="${OLDPWD}"
+  while [ "${path_}" != "" ] && [ ! -f "${path_}/.nvmrc" ]; do
+    path_=${path_%/*}
+  done
+  nvm_echo "${path_}"
+}
```

Comment:
isn't this just `nvm_find_up .nvmrc`?

Reply:
`PWD=OLDPWD nvm_find_up .nvmrc`?

Reply:
To be more accurate, this is a combination of `nvm_find_nvmrc` and `nvm_find_up`.  However, `nvm_find_up` exclusively searches up from the working directory, where here we want to search up from the old working directory.  Technically you are correct, we could overwrite the value in `PWD` with `OLDPWD` before calling `nvm_find_nvmrc`, however overwriting the value of `PWD` to something other than the current working directory just felt wrong.

Reply:
I agree, but it seems better to utilize nvm's existing functionality as much as possible, since it's tested :-)

Reply:
fair enough.  Do you want me to clean up the commits on this PR or will you squash merge?